spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayProperties.java

```diff
@@ -328,6 +329,33 @@
 	 */
 	private Boolean skipExecutingMigrations;
 
+	/**
+	 * The REST API URL of the Conjur server. Requires Flyway teams.
+	 */
+	private String conjurUrl;
+
+	/**
+	 * The Conjur token required to access secrets. Requires Flyway teams.
+	 */
+	private String conjurToken;
+
+	/**
+	 * The REST API URL of the Vault server. Requires Flyway teams.
+	 */
+	private String vaultUrl;
+
+	/**
+	 * The Vault token required to access secrets. Requires Flyway teams.
+	 */
+	private String vaultToken;
+
+	/**
+	 * A comma-separated list of paths to secrets that contain Flyway configurations. Each
+	 * path must start with the name of the engine and end with the name of the secret
+	 * such 'kv/test/1/config'. Requires Flyway teams.
+	 */
+	private String vaultSecrets;
```

Comment:
If that's a comma separated list, it can't be a `String`. We should offer a way for users to also use yaml list if they want to.